src/components/card.jsx

```
@@ -140,24 +132,23 @@
-        <div className="catalog--container--content">
-          {found ? showOneBreed ? renderSelectedBreed() : renderCatCards() : <img src ={noFound} className='nofound'/>}
+    <main className='flex flex-col items-center bg-white shadow-lg rounded-2xl p-5'>
+      <div className="flex flex-col items-center">
+        {found && (
+          <div className="text-center mb-5">
+            <img src={Catalog} className='w-full max-w-md' alt="Catalog Banner" />
+            <p className='text-yellow-500 font-semibold mt-2'><FaInfoCircle className='inline-block mr-1' /> Click on a cat card to know more!</p>
+          </div>
+        )}
+        <div className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 gap-4">
+          {found ? showOneBreed ? renderSelectedBreed() : renderCatCards() : <img src={noFound} className='w-full max-w-md' />}
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)